FlowCrypt/Common/Constants.swift

```diff
@@ -11,6 +11,7 @@ enum Constants {
     // TODO: Anton - Use Themes instead of Constants
     static let NUMBER_OF_MESSAGES_TO_LOAD = 20
 
+    static let inboxCellHeight: CGFloat = 90.0
```

Comment:
please add this to InboxViewController enum Constants. This one is pretty local only to this controller. Shouldn't be in Common constants